addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/callbacks/FritzAhaSetSwitchCallback.java

```diff
@@ -28,14 +28,17 @@
     /**
      * Constructor
      *
-     * @param webIface Interface to FRITZ!Box
-     * @param ain AIN of the device that should be switched
-     * @param switchOn true - switch on, false - switch off
+     * @param webIface
+     *            Interface to FRITZ!Box
+     * @param ain
+     *            AIN of the device that should be switched
+     * @param switchOn
+     *            true - switch on, false - switch off
      */
     public FritzAhaSetSwitchCallback(FritzahaWebInterface webIface, String ain, boolean switchOn) {
         super(WEBSERVICE_PATH, "ain=" + ain + "&switchcmd=" + (switchOn ? "setswitchon" : "setswitchoff"), webIface,
                 Method.GET, 1);
-        this.itemName = ain;

```

Comment:
You are removing `this`'s here, please also remove them in the code you added :-)